packages/ckit/src/tx-builders/ExchangeSudtForCkbBuilder.ts

```diff
@@ -0,0 +1,206 @@
+import { Address, HexNumber } from '@ckb-lumos/base';
+import { CkbTypeScript } from '@ckitjs/base';
+import {
+  Amount,
+  AmountUnit,
+  Builder,
+  Cell,
+  RawTransaction,
+  Transaction,
+  cellOccupiedBytes,
+  WitnessArgs,
+} from '@lay2/pw-core';
+import { BigNumber } from 'bignumber.js';
+import { Pw } from '../helpers/pw';
+import { CkitProvider } from '../providers';
+import { AbstractPwSenderBuilder } from './pw/AbstractPwSenderBuilder';
+
+const minExchangeProviderCkb = new Amount('441', AmountUnit.ckb); // 379 + 61 + 1
+const feeCkb = new Amount('1', AmountUnit.ckb);
+
+export interface ExchangeSudtForCkbOptions {
+  sudt: CkbTypeScript;
+  sudtSender: Address;
+  // exchange sudtAmountForExchange for ckbAmountForRecipient
+  sudtAmountForExchange: HexNumber;
+  sudtAmountForRecipient: HexNumber;
+
+  exchangeProvider: Cell[] | Address;
```

Comment:
> we should use the Lumos's [Cell](https://lumos-website.vercel.app/api/interfaces/base.cell-1.html) here instead of PW-core's Cell. Ckit using Lumos's types for exposed APIs

got it